src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ExportRuleAttribute.cs

```diff
@@ -20,7 +24,7 @@ internal class ExportRuleAttribute : ExportPropertyXamlRuleDefinitionAttribute
         ///     One or more of <see cref="PropertyPageContexts"/>.
         /// </param>
         public ExportRuleAttribute(string ruleName, params string[] contexts)
-            : base(ThisAssembly.FullName, $"XamlRuleToCode:{ruleName}.xaml", string.Join(";", contexts))
+            : base(Assembly.GetExecutingAssembly().FullName, $"XamlRuleToCode:{ruleName}.xaml", string.Join(";", contexts))
```

Comment:
In migrating to Nerdbank.GitVersioning, there is a generation mechanism that conflicted with our own, which created a `.ThisAssembly.cs` file. The only value that was not represented in the NBGV version was this value, `FullName`. There was 2 ways to approach replacing this (as FullName was only referenced one, here in this file). The simplest is to use this reflection call to `GetExecutingAssembly()`. If that isn't performant enough, then I have commented code at the top of the class to create a const string, using the exact same logic we used to generate in our `.ThisAssembly.cs` file.